install/kubernetes/helm/istio/values-istio-demo-common.yaml

```diff
@@ -4,6 +4,14 @@
 # Note: please only put common configuration for the demo profiles here.
 global:
   proxy:
+    init:
```

Comment:
The values yaml file are the defaults so I don't think we need to explicitly add this here, or did you run into an issue without it?

Reply:
I thought this could be the cause of the failed tests but I think that's incorrect. I figured it should be inheriting the defaults. I can revert this commit.

Reply:
Looks like failed test was wrong indentation :cry: my bad

Reply:
Should be `| indent 2`

Reply:
Still failed, will have to look more tomorrow.